processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -28,6 +28,7 @@
 public enum Message {
 
     // CHECKSTYLE:OFF
+// CHECKSTYLE:OFF
```

Comment:
The `CHECKSTYLE:OFF` is needed for keeping the messages on one line (violates the line length). However, I did not touch this file functionally anymore..